tests/foreman/cli/test_discoveredhost.py

```diff
@@ -48,16 +48,45 @@ def test_rhel_pxe_discovery_provisioning(
 
     :BZ: 1731112
     """
+
     sat = module_discovery_sat.sat
     provisioning_host.power_control(ensure=False)
     mac = provisioning_host.provisioning_nic_mac_addr
+    assert sat.execute('systemctl restart dhcpd').status == 0
```

Comment:
Same, Do we really need this workaround now?

Reply:
No. we do not see the issue now of unable to discover the host. I will remove it.